unity/Assets/Maroon/scenes/experiments/ThrowingBalls/scripts/ParameterUI.cs

```diff
@@ -0,0 +1,621 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+using Maroon.UI;
+using ObjectsInUse;
+
+namespace ObjectsInUse
+{
+    public enum ParticleObject
+    {
+        Default,
+        Ball,
+        Satellite
+    }
+}
+
+public class ParameterUI : PausableObject, IResetObject
```

Comment:
for maintenance, this class is a very bad idea
e.g. don't use UI references in the code. use e.g. the onvalue change event to handle the input
please contact me in this case so that we can resolve this issue.

Reply:
I would like to discuss this with you in our next meeting.

Reply:
As we discussed today (10.11.21): We will leave the ParameterUI class as it is for the moment and create a future-issue to refactor this class in the future.